README.md

```diff
@@ -30,92 +29,100 @@ Or install it yourself as:
+    render json: @car, status: :ok
+  end
 
-##### Index
-check if status ok 
+  def destroy
+    @car.destroy!
+  end
 
-************************************************************************************
-### Factory
-The factory will be generated with it's custom generator, defined in the instalation of the factory-bot
+  private
 
-************************************************************************************
-### Routes
-the routes generated are the basic resources routes:
-    table_name: resources
+  def set_car
+    @car = Car.find(params[:id])
+  end
 
-************************************************************************************
-### Scope
-It's possible to add flag in the command to generate an endpoint with a parent
-   $ rails g generator table_name attributes --scope father_name
+  def car_params
+    params.require(:car).permit(:name, :color)
+  end
+end
 
-the only thing this won't genrate it's the nested routes, but it will generate all the previous files mentioned just liek a scaffold
+```
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can
+also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and the created tag, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the
+version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version,
+push git commits and the created tag, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/rest-api-generator. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [code of conduct](https://github.com/[USERNAME]/rest-api-generator/blob/master/CODE_OF_CONDUCT.md).
+Bug reports and pull requests are welcome on GitHub at https://github.com/SwitchDreams/rest-api-generator. This project
+is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to
+the [code of conduct](https://github.com/SwitchDreams/rest-api-generator/blob/master/CODE_OF_CONDUCT.md).
 
 ## License
 
 The gem is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
 
 ## Code of Conduct
```

Comment:
Acho que pode tirar essa parte a não ser que o code of conduct seja melhor explicado, pq já tem la em cima a mesma coisa